rest/handler.go

```diff
@@ -201,6 +203,17 @@ func (h *handler) invoke(method handlerMethod) error {
 		}
 	}
 
+	// If an Admin Request we need to check the user credentials
+	// FIXME: Temporarily disable this
+	if false {
+		clusterAddress, _, _, _, _, _ := tempConnectionDetailsForManagementEndpoints()
+		if !base.ServerIsWalrus(clusterAddress) && ((h.privs == adminPrivs && *h.server.config.AdminInterfaceAuthentication) || (h.privs == metricsPrivs && *h.server.config.MetricsInterfaceAuthentication)) {
```

Comment:
It doesn't feel right to have a check for walrus here - it's more like we should default the InterfaceAuthentication config options to false when loading the bootstrap config.

Reply:
Agreed but currently there is no way to nicely do this. Talked this through with Ben a little and he has something similar doing an initial check for walrus with his persistent config work so I'll probably want to leave this temporary check in `shouldCheckAdminAuth` for now and this can be replaced later. Might add this to the `Update tempConnectionDetails() to use bootstrap credentials on ServerContext` ticket so we remember to do this if we're happy keeping it in here.